src/nft/components/details/AssetDetails.css.ts

```diff
@@ -23,6 +23,39 @@ export const image = style([
   },
 ])
 
+export const embedContainer = style({
+  position: 'relative',
+  overflow: 'hidden',
+  width: '100%',
+  paddingTop: '100%',
+})
+
+export const embed = style({
```

Comment:
you can import the styles from `image` directly to avoid a lot of the duplicated code here and help maintain consistency

Reply:
@cbachmeier good call, will update!